utils/docker/Dockerfile.centos.7

```diff
@@ -68,11 +68,21 @@ RUN yum -y install \
            python2-avocado python2-avocado-plugins-output-html \
            python2-avocado-plugins-varianter-yaml-to-mux       \
            yum-plugin-copr python-pathlib                      \
-           python2-clustershell python2-Cython python2-pip;    \
+           python2-clustershell python2-Cython python2-pip     \
+           python36-clustershell python36-paramiko             \
+           python3-pip;                                        \
     yum -y copr enable jhli/ipmctl;                            \
     yum -y copr enable jhli/safeclib;                          \
     yum -y install libipmctl-devel
 
+# DAOS python 3 packages required for pylint
+#  - excluding mpi4py as it depends on CORCI-635
+RUN pip3 install avocado-framework; \
+    pip3 install avocado-framework-plugin-result-html; \
+    pip3 install avocado-framework-plugin-varianter-yaml-to-mux; \
+    pip3 install numpy; \
+    pip3 install jira
```

Comment:
I don't see where a python2 of this is being installed.

Reply:
It is a requriremnt of the `python2-avocado-plugins-varianter-yaml-to-mux` rpm.

Reply:
I can get it from the python36-jira rpm instead

Reply:
> > Yes, the file that invokes pylint, `utils/check_modules.sh`, passes in the files from the src directory.
> 
> And this all very much makes sense. We lint _source_ code not installed product. Moreover, we lint before we even start building so we have to be linting the source.

Agreed.  Now I think about this more I remember commenting on it at the time, which is possibly what lead to this ticket being filed.